administrator/components/com_associations/language/en-GB/en-GB.com_associations.ini

```diff
@@ -29,6 +29,11 @@ COM_ASSOCIATIONS_HEADING_MENUTYPE_ASC="Menu ascending"
 COM_ASSOCIATIONS_HEADING_MENUTYPE_DESC="Menu descending"
 COM_ASSOCIATIONS_NO_ASSOCIATION="There is no association for this language"
 COM_ASSOCIATIONS_NOTICE_NO_SELECTORS="Please select an Item Type and a reference language to view the associations."
+COM_ASSOCIATIONS_PURGE="Clear all associations"
+COM_ASSOCIATION_PURGE_CONFIRM_PROMPT="Are you sure you want to clear all associations? Confirming will permanently delete them!"
+; Do not translate the name of the table
+COM_ASSOCIATIONS_PURGE_FAILED="Failed to clear the 'associations' table."
+COM_ASSOCIATIONS_PURGE_SUCCESS="The 'associations' table was successfully cleared."
```

Comment:
I do agree though that it is not consequent in core

![screen shot 2016-08-12 at 10 43 44](https://cloud.githubusercontent.com/assets/869724/17617313/bc56dac8-6079-11e6-8543-bd36d85cda1d.png)